### PR TITLE
checking for errors

### PR DESCRIPTION
**Pull request by Kidd254**

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Kidd254, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that has no pull request description and a title not descriptive enough. Kindly always provide adequate PR title and description for your PRs in order to make it easier to know the changes applied at a glance.

Also, you seem to have made most of your changes to the main branch instead of a feature branch as recommended for GitHub flow. Kindly make sure that your PR contains all the changes made in the milestone in order to allow for a reviewer to make comments on the changes.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -12,64 +12,56 @@
...
-        <circle cx="60.5" cy="548.5" r="7" fill="white" stroke="#CBCDFF" />
-        <path
-            d="M421.241 74.9701C417.195 67.7876 411.514 61.659 404.659 57.0803C397.803 52.5017 389.966 49.6018 381.782 48.6154C373.597 47.6291 365.295 48.584 357.548 51.4028C349.802 54.2216 342.827 58.8252 337.191 64.8407C331.554 70.8562 327.413 78.1145 325.103 86.0281C322.793 93.9416 322.379 102.288 323.895 110.391C325.411 118.494 328.813 126.127 333.828 132.67C338.842 139.214 345.326 144.485 352.756 148.056L362.543 127.695C358.31 125.661 354.616 122.658 351.759 118.93C348.903 115.202 346.964 110.854 346.101 106.238C345.237 101.621 345.473 96.8662 346.789 92.3579C348.105 87.8496 350.464 83.7145 353.675 80.2874C356.886 76.8604 360.86 74.2378 365.273 72.6319C369.686 71.026 374.416 70.482 379.079 71.044C383.741 71.6059 388.206 73.258 392.112 75.8664C396.017 78.4748 399.254 81.9663 401.559 86.0582L421.241 74.9701Z"
-            fill="#6070FF" />
-        <rect x="294" y="61.79" width="95.5804" height="22.021" transform="rotate(-30 294 61.79)" fill="#EBEBFF" />
-    </svg>
-</body>
-<div class="header">I'm Lawrence Glad to see you</div>
```
Comment (Nyame-Wolf) [inline index.html:42]: Kindly use a `<h1>` element for the line I'm Lawrence
then use `<h2>` or any other header tags.

The div element is too general and this header is  used for the title of your file.
You can read more [here](https://www.reliablesoft.net/h1-tag/)

File: index.html [hunk 2]
```diff
@@ -12,64 +12,56 @@
...
-        fill="#505F79" />
-    <path
-        d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
-        fill="#505F79" />
-</svg>
+
```
Comment (Nyame-Wolf) [inline index.html]: Please have all this in the css file.

File: index.html [hunk 3]
```diff
@@ -12,64 +12,56 @@
...
+    <div class="background">
+        <div class="toolbar enabled">
+            <h1 class="logo"> My Logo </h1>
```
Comment (Nyame-Wolf) [inline index.html]: Kindly note that this should be a link to use an anchor element. 
This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 4]
```diff
@@ -12,64 +12,56 @@
...
+                        fill="#505F79" />
+                    <path
+                        d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
+                        fill="#505F79" />
+                </svg>
+            </div>
```
Comment (Nyame-Wolf) [inline index.html]: Kindly note that this should use links. `<ul>` and `<li>`
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

File: index.html [hunk 5]
```diff
@@ -12,64 +12,56 @@
...
-        <circle cx="303.5" cy="216.5" r="5" fill="white" stroke="#CBCDFF" />
-        <circle cx="60.5" cy="548.5" r="7" fill="white" stroke="#CBCDFF" />
-        <path
-            d="M421.241 74.9701C417.195 67.7876 411.514 61.659 404.659 57.0803C397.803 52.5017 389.966 49.6018 381.782 48.6154C373.597 47.6291 365.295 48.584 357.548 51.4028C349.802 54.2216 342.827 58.8252 337.191 64.8407C331.554 70.8562 327.413 78.1145 325.103 86.0281C322.793 93.9416 322.379 102.288 323.895 110.391C325.411 118.494 328.813 126.127 333.828 132.67C338.842 139.214 345.326 144.485 352.756 148.056L362.543 127.695C358.31 125.661 354.616 122.658 351.759 118.93C348.903 115.202 346.964 110.854 346.101 106.238C345.237 101.621 345.473 96.8662 346.789 92.3579C348.105 87.8496 350.464 83.7145 353.675 80.2874C356.886 76.8604 360.86 74.2378 365.273 72.6319C369.686 71.026 374.416 70.482 379.079 71.044C383.741 71.6059 388.206 73.258 392.112 75.8664C396.017 78.4748 399.254 81.9663 401.559 86.0582L421.241 74.9701Z"
-            fill="#6070FF" />
-        <rect x="294" y="61.79" width="95.5804" height="22.021" transform="rotate(-30 294 61.79)" fill="#EBEBFF" />
-    </svg>
```
Comment (Nyame-Wolf) [inline index.html:40]: This is a tremendous job you have done setting up your the background images. You have used the inline styles for the shapes. I recommend using the css file for all the css related information. Read [here](https://www.hostinger.com/tutorials/difference-between-inline-external-and-internal-css#:~:text=Advantages%20of%20External%20CSS%3A,css%20file%20for%20multiple%20pages.) why this is the recommended format

File: index.html [hunk 6]
```diff
@@ -12,64 +12,56 @@
...
-</body>
```
Comment (Nyame-Wolf) [inline index.html:41]: This should appear at the end of the page, just before the closing `<html>` tag

File: index.html [hunk 7]
```diff
@@ -12,64 +12,56 @@
...
-        fill="#505F79" />
-    <path
-        d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
-        fill="#505F79" />
-</svg>
+
+    <div class="background">
```
Comment (Nyame-Wolf) [inline index.html]: Please use a `<section>` tag here.
It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @Kidd254  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the same font as Figma design ☑️ 
- Commits are descriptive and meaningful :+1: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

- [ ] Kindly note that your submission does not meet figma template requirements. has a fixed width and height and should not It also has margins that are not similar to the figma template's
- [ ] Your design
Screenshot 2022-09-23 at 13 25 24

- [ ] Figma template:
Screenshot 2022-09-23 at 12 35 53

Please match the design.

 

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by emmiiorji — APPROVED** [review]
Hi @Kidd254, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations


## Good Highlights✔

- Adherence to GitHub flow
- Good commit messages
- Requested changes have been implemented

## Optional suggestions

- I suggest that you endeavor to always provide a description to your PR in order tell at a glance what changes has occurred.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**